lib/utils/contentSearch.ts

```diff
@@ -208,6 +211,32 @@ export const searchContentWithReactions = async (options: ContentSearchOptions):
   };
 };
 
+/**
+ * Load comments for all content results in background
+ * This is called after search results are displayed to improve UX
+ * @param results - Array of search results to load comments for
+ */
+export const loadCommentsForSearchResults = async (results: SearchResult[]): Promise<void> => {
+  try {
+    // Extract event IDs from content results
+    const contentEventIds = results
+      .filter(result => result.type === 'content')
+      .map(result => result.event.id);
+
+    if (contentEventIds.length === 0) return;
+
+    console.log(`[CONTENT-SEARCH] Loading comments for ${contentEventIds.length} content results in background`);
+
+    const { loadCommentCountsFromRelays } = await import('./commentUtils');
```

Comment:
Better to import at the top level unless it's a heavy dependency of you're trying to optimize code splitting